Jenkinsfile_nightly

```diff
@@ -47,5 +47,7 @@ withNightlyPipeline(type, product, component) {
 
     enableMutationTest()
     enableFullFunctionalTest()
-    enableSecurityScan()
+
+    //This is not required as this is batch application and we don't have any API
+    //enableSecurityScan()
 }
```

Comment:
securityscan is not limited to APis ... it should be done for both

Reply:
Its not a batch application

Reply:
is communicated by testing team